server/services/gameLogic.js

```diff
@@ -113,6 +142,8 @@ exports.playCardAndWord = (username, cardId, word) => {
             cardId: cardId
         };
         cards.push(card);
+        let playerCards =  players.find(player => player.username === currentPlayer.username).cards;    
+        playerCards.find(playedCard => playedCard.id.toString() === cardId).played = true;
```

Comment:
I don't think this does anything

Reply:
The return from find will be the actual object, so it should work.